sklearn/linear_model/_stochastic_gradient.py

```diff
@@ -824,18 +830,14 @@ class SGDClassifier(BaseSGDClassifier):
     learning_rate : str, default='optimal'
         The learning rate schedule:
 
-        'constant':
-            eta = eta0
-        'optimal': [default]
-            eta = 1.0 / (alpha * (t + t0))
-            where t0 is chosen by a heuristic proposed by Leon Bottou.
-        'invscaling':
-            eta = eta0 / pow(t, power_t)
-        'adaptive':
-            eta = eta0, as long as the training keeps decreasing.
-            Each time n_iter_no_change consecutive epochs fail to decrease the
-            training loss by tol or fail to increase validation score by tol if
-            early_stopping is True, the current learning rate is divided by 5.
+        - 'constant': `eta = eta0`
+        - 'optimal': `eta = 1.0 / (alpha * (t + t0))`
+          where t0 is chosen by a heuristic proposed by Leon Bottou.
+        - 'invscaling' (default): `eta = eta0 / pow(t, power_t)`
```

Comment:
The default is "optimal" for SGDClassifier, and "invscaling" for SGDRegressor.

Reply:
good catch, thanks